### ISSUE TITLE
LAVIS CLIP

### ISSUE THREAD

**Issue by StrongLachlanM** [StrongResearch/isc-demos#66]
Source / repo
https://github.com/salesforce/LAVIS

Model description
[DESCRIPTION]

Dataset
[DATASET]

Literature benchmark source
[URL]

Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

Strong Compute result achieved
[VALUE/S]

Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

Logs gist
[URL]
